package.json

```diff
@@ -93,12 +101,12 @@
     "output-file-sync": "^1.1.1",
     "phantomjs-prebuilt": "^2.1.7",
     "portfinder": "^1.0.3",
-    "react": "^15.0.0",
-    "react-component-metadata": "^2.1.1",
-    "react-dom": "^15.0.0",
+    "react": "^15.0.1",
+    "react-component-metadata": "^3.0.0",
```

Comment:
hmm that doesn't seem to explicity be about metadata...

I have a branch with half of babel6 support in it let me try that and see if I get something similar